packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -0,0 +1,159 @@
+- A local application. Or use the one provided in the following guide.
+- The ngrok utility. Installation details are provided in the following guide.
+
+With those prerequisites satisfied, use the following steps to get up-and-running:
+
+1. [Install ngrok](/#install-ngrok)
+1. [Create a local sample app](/#create-a-local-application)
+1. [Set up an Event Hook](/#create-an-okta-event-hook)
+1. [Preview and test](/#preview-test-and-review-the-event-hook)
+
+## Install ngrok
+
+If you already have ngrok installed, move on to the next step! If not, follow the installation instructions at [https://ngrok.com/download](https://ngrok.com/download).
+
+Some installation notes:
+
+- You do not need to have an account to install and run ngrok, but creating a free account provides more features as well as basic authentication.
+- You can install ngrok directly in your project folder, as documented from the ngrok download page. Or you can install on your system's path directory to be able to run ngrok from any folder. Alternatively, you can install the executable in your favorite local folder, but you'll need the folder path when referencing the tool.
+
+### Run ngrok
+
+After installing ngrok, ensure that it's running by creating a "tunnel" into a local port (8082 in this example). If you installed directly into your project folder, run the following command in your terminal:
+
+```terminal
+> ./ngrok http 8082
+```
+
+or if you installed in your system path:
+
+```terminal
+> ngrok http 8082
+```
+
+or if you used your favorite folder:
+
+```terminal
+> ~/applications/ngrok http 8082
+```
+
+If you see the following content in your terminal, ngrok is running successfully:
+
+![A screen shot of a terminal that displays an ngrok session status, with online in green. The session status contains urls that tunnel into the local port.](/img/ngrok-and-event-hooks-session-status.png)
+
+See [ngrok](https://ngrok.com) or their [documentation](https://ngrok.com/docs) for further information.
+
+## Create a local application
+
+<StackSelector snippet="sample-app"/>
+
+## Create an Okta Event Hook
+
+Create the Okta Event Hook to work with your local application, which can now be exposed externally. The Event Hook must be set up and verified within your Okta Admin Console.
+
+### Set up the Event Hook
+
+1. Sign in to your [Okta org](https://login.okta.com/).
+
+2. From the Admin Console, go to **Workflow** > **Event Hooks**.
+
+3. Click **Create Event Hook**. The **Add Event Hook Endpoint** dialog box opens.
+
+4. In the **Name** field, add a unique name for the Hook (in this example, "Deactivated User Event Hook").
+
+5. In the **URL** field, add your external service URL, including endpoint. For this example, use the code endpoint, `/userDeactivated` from `server.js` with the `https://` URL from the ngrok session. For example, your URL should appear similar to: `https://28333dd3ddf3.ngrok.io/userDeactivated`.
```

Comment:
I had to scroll up to the ngrok session screenshot to figure out what my ngrok URL was - how we can make this a bit more clear on where to find this for those unfamiliar with ngrok and using it for the 1st time. (See my earlier comment on highlighting the ngrok URL)

Reply:
I think I'll add another snippet screen shot of the ngrok terminal, just focusing on the URL. That should be enough for comprehension, I hope.